test/domain-shared/git-utils.test.js

```diff
@@ -0,0 +1,65 @@
+describe("GitUtils", () => {
+    const scenarioModifiedOutput = {
+        modified: [
+            "package-lock.json",
+            "package.json",
+            "server_src/lib/serverCommands.js",
+        ],
+        ahead: 0,
+        behind: 0,
+    };
+
+    let wasReset = null;
+
+    jest.mock("simple-git", () => {
```

Comment:
Similar to system-command tests, but I'd suggest to keep a copy locally as each test is different.